src/pages/DisplayFixtures.jsx

```diff
@@ -122,14 +112,16 @@ const DisplayFixtures = () => {
           },
           body: JSON.stringify(update),
         });
-        setAllPredictions([...predictions, prediction]);
-        setEditMode(false);
         // console.log(`Predictions updated in database: ${JSON.stringify(predictions)}`)
-
       } catch (error) {
         console.error('Failed to create/update listing:', error);
       };
     };
+    const fetchedPredictions = await getPredictions();
+    setAllPredictions(fetchedPredictions);
+    setChangedPredictions([]);
```

Comment:
remember to reset your `changedPredictions` once all the changes have been written to DB so that we don't call redundant `PUT` APIs for these updated predictions in the future!